src/System.Text.RegularExpressions/tests/RegexCompilationInfoTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Text.RegularExpressions.Tests
+{
+    [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot)]
+    public class RegexCompilationInfoTests
+    {
+        private RegexCompilationInfo Instance
+        {
+            get => new RegexCompilationInfo("pattern", RegexOptions.None, "name", "fullnamespace", true, TimeSpan.FromSeconds(5));
```

Comment:
Given the range of slowness we see in the test runs, and other tests that can run concurrently, I wonder whether timeouts should either be negligible (so they always fire) or large (so they only fire if the test is truly hung)

I've seen other tests where the timeout was originally 1 sec or 2 sec and we progressively increased it to 30 sec just to avoid occasional timeouts.